src/components/wallet-manager/index.tsx

```diff
@@ -38,6 +39,7 @@ export class Container extends React.Component<Properties, State> {
       currentAddress: value.address,
       connectionStatus: status,
       isWalletModalOpen,
+      isAuthenticated: !!state.authentication.user?.data && !!value.address,
```

Comment:
Is it possible to get `isAuthenticated` from the context rather than the state in this situation? [context: { isAuthenticated }](https://github.com/zer0-os/zOS/pull/195/files#diff-ba9e5f36913253ad75371769e2dcf1ca405991e16351de5ce97f7394ff8a756eR20)

Reply:
I tried to get the `isAuthenticated` from the context but it is not possible since in the channel the context is used from the `AuthenticationContextProvider` added in the container of the channels app.